roles/openshift_node_bootstrap_configmap/tasks/standard.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Why did we make this change?  The previous format was better, because there's nothing special about infra or compute groups.   This also broke GCE because the masters are a bootstrap node group.

I'd like to revert this and go back to a format where we have list of groups with a list of edits.  We aren't going to just have 3 stock groups, we're going to have sets of configurations defined by the master.

Reply:
We should not be treating node groups as concrete things - they're a fungible list in the config.  We will be dynamically adding and removing node groups.  Infra is likely to be broken into several node groups, some of which are autoscaling and some of which are not.